executor/executor_test.go

```diff
@@ -6714,6 +6714,77 @@ func (s *testCoprCache) TestIntegrationCopCache(c *C) {
 	c.Assert(hitRatio > 0, Equals, true)
 }
 
+func (s *testSerialSuite) TestCoprocessorOOMTicase(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec(`set @@tidb_wait_split_region_finish=1`)
+	// create table for non keep-order case
+	tk.MustExec("drop table if exists t5")
+	tk.MustExec("create table t5(id int)")
+	tk.MustQuery(`split table t5 between (0) and (10000) regions 10`).Check(testkit.Rows("9 1"))
```

Comment:
The number of regions is 10, it is too small.  All task will be deal with in the first round.